bin/tat

```diff
@@ -2,7 +2,8 @@
 #
 # Attach or create tmux session named the same as current directory.
 
-session_name="$(basename "$PWD" | tr . -)"
+path_name="$(basename "$PWD" | tr . -)"
+session_name=${1-$path_name}
```

Comment:
@gabebw I'm using this form in my dotfiles, and it's been working fine (I'm using ZSH).

what do the `:` and quotes add?